scanner/config.go

```diff
@@ -168,6 +168,33 @@ func runScanners(scannerList []scanners.ScannerInterface, searchDir string) map[
 	return scannerOutputs
 }
 
+func initOptionTypeRecursive(option *models.OptionNode) {
```

Comment:
An alternative solution suggested is to set the default type selector when creating an option with NewOption. Manually Remove all occurences of "_" used as a key in optionNode map. (Besides the usages for fastlane project type (toolscanner), and check that also works after the changes)